spec/helpers/schema_helper_spec.rb

```diff
@@ -486,116 +486,6 @@
       end
     end
 
-    describe "product_details" do
-      it "allows product_details to be blank" do
-        data = valid_data.except(:product_details)
-        expect(validate_against_form_response_schema(data)).to be_empty
-      end
-
-      it "doesn't return error when medical_equipment_type is missing" do
-        data = valid_data.tap do |valid_data|
-          valid_data[:product_details].last.delete(:medical_equipment_type)
-        end
-
-        expect(validate_against_form_response_schema(data).first).to be(nil)
-      end
-
-      it "returns a list of errors when medical_equipment_type has an unexpected value" do
-        data = valid_data.tap do |valid_data|
-          valid_data[:product_details].last[:medical_equipment_type] = "Foo"
-        end
-
-        expect(validate_against_form_response_schema(data).first).to include("medical_equipment_type")
-      end
```

Comment:
These tests for `medical_equipment_type should be kept and moved out of the "product_details" block as we're still asking the question.